### DISCUSSION TITLE
✨ pyLoad

### DISCUSSION THREAD

**Discussion by untitled303** [runtipi/runtipi-appstore#4906] [Requests]
### Description

Free and Open Source download manager written in Python and designed to be extremely lightweight, easily extensible and fully manageable via web

### Official Website

https://pyload.net/

### Documentation link

https://github.com/pyload/pyload/wiki

### Last application release & date

0.4.20 (20.02.2020)

### Application license

GNU Affero General Public License

### Source code repository link

https://github.com/pyload/pyload

### Docker image link

https://github.com/linuxserver/docker-pyload-ng/

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by Raiper34** [comment]
I created PR for it https://github.com/runtipi/runtipi-appstore/pull/5682
